packages/components/pie-toast/src/defs.ts

```diff
@@ -61,10 +74,12 @@ export const ON_TOAST_OPEN_EVENT = `${componentSelector}-open`;
  */
 export const ON_TOAST_LEADING_ACTION_CLICK_EVENT = `${componentSelector}-leading-action-click`;
 
-export type DefaultProps = ComponentDefaultPropsGeneric<ToastProps, 'isOpen' | 'isDismissible' | 'isMultiline'>;
+export type DefaultProps = ComponentDefaultProps<ToastProps, 'isOpen' | 'isDismissible' | 'isMultiline' | 'variant' | 'isStrong'>;
```

Comment:
Up to you, but with this alternative syntax, if someone adds a new prop it will force them to either add a new default value or add it to the list of props to omit. What do you reckon?
```suggestion
export type DefaultProps = ComponentDefaultProps<ToastProps, Omit<ToastProps, 'message' | 'leadingAction'>>;
```

Reply:
That's a nice idea! The suggestion above returns the error below, but I managed to apply this solution by using the following solution.
```
export type DefaultProps = ComponentDefaultProps<ToastProps, keyof Omit<ToastProps, 'message' | 'leadingAction'>>;
```
Error:
![Screenshot 2024-07-26 at 1 16 00 PM](https://github.com/user-attachments/assets/bb48fd64-ad04-4208-941e-9b0b08fd9778)

Reply:
Ah yes, I always forget the `keyof`!